lib/r10k/action/deploy/display.rb

```diff
@@ -1,30 +1,10 @@
-require 'r10k/util/setopts'
 require 'r10k/deployment'
-require 'r10k/logging'
+require 'r10k/action/base'
 
 module R10K
   module Action
     module Deploy
-      class Display
-
-        include R10K::Util::Setopts
-        include R10K::Logging
-
-        def initialize(opts, argv)
-          @opts = opts
-          @argv = argv
-          setopts(opts, {
-            :config     => :self,
-            :puppetfile => :self,
-            :detail     => :self,
-            :format     => :self,
-            :fetch      => :self,
-            :trace      => :self
-          })
-
-          @level  = 4
```

Comment:
I think those were used for the old way the output was formatted and I probably should have removed them when I converted to yaml/json output.